onc-healthit/inferno-program
# FI-1321: Fix Must Support tests incorrectly passing

# Summary
Addresses #355. The bulk group validation sequence was modifying the US Core profile definitions. Any MS field found in that sequence would not be checked in any future test runs.

## Code changes

- US Core profile definitions are now deep frozen, so any attempt to modify them will raise an error
- The bulk group validation sequence now deep dups the profile definitions so that the originals will not be modified

## Testing guidance

- Run the single patient API tests with patient 85. USCDR-13 should skip.
- Run the bulk tests
- Run the single patient API tests with patient 85 again. USCDR-13 should still skip. If you follow these three steps on main, it will pass.
